packages/server/playground.html

```diff
@@ -45,6 +45,12 @@
           documentUrl: `${EXAMPLES_HOST_URL}/weather.html`,
           title: "Weather API",
         },
+        // {
```

Comment:
Looks like this wasn't meant to be left in?

Reply:
Removed in [commit 5](https://github.com/mml-io/mml-playground/pull/21/commits/01fc1ea646d4c1e9d0eb55bcff3e2de0202b47ff#diff-dd71c845ca491c6b6cc9a30162ab15b1d5f020138a7c58e8e9ba18cccea293e0L48)